server/edd/campaign/templates/edd/campaign/detail.html

```diff
@@ -41,7 +41,6 @@
       </form>
     </div>
   {% endif %}
-  <h1>{{ campaign.name }}</h1>
```

Comment:
Why removing this? The pages aren't active yet, but there should be a heading here.

Reply:
I replaced this in the next commit.